### ISSUE TITLE
Document xxf:dynamic

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#2689]

**Comment by ebruchez** [comment]
Yes would be good to document, especially that in 4.11 it's working outside of the Form Builder main area and is used for [custom control settings](http://blog.orbeon.com/2016/03/flexible-custom-control-settings.html). To document:

- what it does
- optimized cases: external updates to instances and XBL content
- stable ids?
- limitations: loading of new JS/CSS (#55), possible switch/repeat state issues (#1898)
